src/test/java/uk/gov/hmcts/juror/job/execution/jobs/contentstore/ContentStoreFileJobTest.java

```diff
@@ -275,15 +285,24 @@ void updateSqlException() throws IOException, SQLException {
                 FileSearch fileSearch = mock(FileSearch.class);
                 fileSearchMock.when(() -> FileSearch.directory(ftpDirectory, true)).thenReturn(fileSearch);
 
-                Set<File> files = new HashSet<>(Set.of(mock(File.class), mock(File.class), mock(File.class)));
+                File file1 = Mockito.mock(File.class);
+                Mockito.when(file1.getName()).thenReturn("sample1.txt");
```

Comment:
Does not need to be fully qualified can just be when(...)

Reply:
Removed